packages/select/src/MultiSelect/MultiSelect.tsx

```diff
@@ -40,7 +40,7 @@ const MultiSelect = <T extends object>(props: MultiSelectProps<T>) => {
   const filterInputRef = useRef<HTMLInputElement>(null);
   const triggerRef = useRef<HTMLButtonElement>(null);
   const popoverRef = useRef<HTMLDivElement>(null);
-  const listBoxRef = useRef<HTMLUListElement>(null);
+  const listBoxRef = useRef<HTMLDivElement>(null);
```

Comment:
I checked with a screenreader to confirm that changing the list from `ul/li` to `div`s doesn't affect accessibility / voiceover output, since we have the correct roles on the list and list items.

Reply:
Bonus points: record a little loom and share it in #ask-frontend.